src/flotilla/agent/db.py

```diff
@@ -23,13 +24,14 @@ class FlotillaAgentDynamo(object):
     """
 
     def __init__(self, instance_id, service_name, status_table,
-                 assignments_table, revisions_table, units_table):
+                 assignments_table, revisions_table, units_table, kms):
```

Comment:
Shouldn't https://github.com/thepwagner/flotilla/blob/master/src/agent.py#L43 also be init'd with KMS now, like https://github.com/pebble/flotilla/pull/10/files#diff-8d6ef5887842a4613dc91cfeefc9f28bR24